files/build_templates/init_cfg.json.j2

```diff
@@ -3,7 +3,8 @@
         "localhost": {
             "buffer_model": {% if default_buffer_model == "dynamic" %}"dynamic"{% else %}"traditional"{% endif %},
             "default_bgp_status": {% if shutdown_bgp_on_start == "y" %}"down"{% else %}"up"{% endif %},
-            "default_pfcwd_status": {% if enable_pfcwd_on_start == "y" %}"enable"{% else %}"disable"{% endif %}
+            "default_pfcwd_status": {% if enable_pfcwd_on_start == "y" %}"enable"{% else %}"disable"{% endif %},
+            "synchronous_mode": {%- if include_p4rt == "y" %}"enable"{% else %}"disable"{% endif %}
```

Comment:
>else [](http://example.com/codeflow?start=72&length=4)

elsewise, can we remove this line completely, so it will be same as before?

Reply:
The comma makes it tricky, what do you think of the suggestion below?

Reply:
I think @qiluo-msft  suggestion was to remove the line completely?

Reply:
To be clear, I suggest 
1. if include_p4rt, set synchronous_mode enable
2. if not, remove the whole line completely.
I agree `,` is tricky.

Reply:
@bocon13, @qiluo-msft , is this comment resolved?

Reply:
I think the easiest fix will be to make this not the last line. That should make things a little cleaner. I’ll push that change and this should be resolved.

Reply:
```suggestion
```

Reply:
@rhalstea what do you think?

Reply:
SGTM

Reply:
@rlhui, I think we have a clean solution that resolves this now